vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewState.kt

```diff
@@ -70,6 +61,15 @@ enum class OnboardingFlow {
     SignInSignUp
 }
 
+@Parcelize
+data class SelectedHomeserverState(
```

Comment:
the individual homeserver state has been bundled into a single `SelectedHomeserverState`, this object is created/updated when starting the authentication flow (by selecting or editing a homeserver url) through a single entry point [StartAuthenticationFlowUseCase.kt](https://github.com/vector-im/element-android/pull/5691/files#diff-bdf9149f4c1ac2c088d3b7f61840fc28eac2579a5d45144b177dc87b741ff04d)